src/misc.c

```diff
@@ -207,14 +209,11 @@ void splitc(char *first, char *rest, char divider)
     memmove(rest, p + 1, strlen(p + 1) + 1);
 }
 
-/*    As above, but lets you specify the 'max' number of bytes (EXCLUDING the
- * terminating null).
+/* As above, but lets you specify the 'max' number of bytes
  *
  * Example of use:
  *
- * char buf[HANDLEN + 1];
- *
- * splitcn(buf, input, "@", HANDLEN);
+ * splitcn(buf, input, "@", sizeof buf);
```

Comment:
do you mean `'@'` instead of `"@"`?

Reply:
why is compiler silent about this? they should have made a call of a function with prototype `char` called with a `char*` an error with c23 instead of destroying our mod API :P

Reply:
ah its in comment ;P

Reply:
fixed.